config/database.php

```diff
@@ -130,3 +130,23 @@
     ],
 
 ];
+
+// If the app is deployed on Heroku interpret the postgres URL
```

Comment:
We need to document this slightly better I think. Like exactly why this is bad and why we do this instead. The reason youve posted on the PR is good enough to me.